plugins/funder.c

```diff
@@ -864,6 +999,30 @@ static char *amount_option(const char *arg, struct amount_sat *amt)
 	return NULL;
 }
 
+static char *option_lease_fee_base(const char *arg,
+				   struct funder_policy *policy)
+{
+	if (!policy->ad)
+		policy->ad = liquidity_ad_default(policy);
+	return amount_option(arg, &policy->ad->lease_base_sat);
+}
+
+static char *option_lease_fee_basis(const char *arg,
+				    struct funder_policy *policy)
+{
+	if (!policy->ad)
+		policy->ad = liquidity_ad_default(policy);
+	return u16_option(arg, &policy->ad->lease_basis);
+}
+
+static char *option_lease_weight_max(const char *arg,
+				     struct funder_policy *policy)
+{
+	if (!policy->ad)
+		policy->ad = liquidity_ad_default(policy);
+	return u16_option(arg, &policy->ad->funding_weight_max);
+}
```

Comment:
This seems to make option behaviour depend on ordering? I think you need to stash these values and override policy after parsing complete

Reply:
I don't see how the ordering matters? Whichever one gets called first will populate the `policy->ad` with the default values, and then update the default to whatever was passed in.

The order shouldn't matter, first one to get called should populate the `ad`, if none gets called then it'll remain NULL.

Reply:
Then why not set policy->ad before parsing?